<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReturnOrderDialog</class>
 <widget class="QDialog" name="ReturnOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Return Order</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>RETURN ORDER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="searchGroup">
     <property name="title">
      <string>Enter Receipt ID</string>
     </property>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLabel" name="receiptIDLabel">
        <property name="text">
         <string>Receipt ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="receiptIDEdit">
        <property name="placeholderText">
         <string>Enter receipt ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>SEARCH</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="receiptGroup">
     <property name="title">
      <string>Receipt Information</string>
     </property>
     <layout class="QVBoxLayout" name="receiptLayout">
      <item>
       <widget class="QLabel" name="receiptInfoLabel">
        <property name="text">
         <string>No receipt loaded</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirmReturnButton">
       <property name="text">
        <string>CONFIRM RETURN</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
